src/js/tabs/configuration.js

```diff
@@ -642,6 +642,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             serialRXtypes.push('FrSky FPort');
         }
 
+        if (semver.gte(CONFIG.apiVersion, "1.41.0"))  {
```

Comment:
The firmware part is planed for version 4.1 of Betaflight. This will correspond, if nothing changes, with MSP 1.42.0.